src/plugins/intel_gpu/src/graph/compilation_context.cpp

```diff
@@ -64,11 +67,7 @@ class CompilationContext : public ICompilationContext {
         _stop_compilation = true;
```

Comment:
Can we call here `remove_keys` for all remaining `_task_keys`? Just to prevent execution of all previously enqueued tasks when it's not really needed anymore